apps/examples/lcd_test/Kconfig

```diff
@@ -0,0 +1,14 @@
+#
```

Comment:
The commit list of this PR appear to be not good..
We usually merge the PR using rebase option.
So Please update base commit of this pr branch into master branch latest commit. 

```
st7789lcd with spi
modified files with conflicts
...
Merge branch 'Samsung:master' into spi-lcd -----> do not create merge commit
...
Merge branch 'spi-lcd' of github.com:anjana348/TizenRT into spi-lcd -----> do not create merge commit
...
```

Reply:
@anjana348 When you update your branch with latest public, you should use git rebase instead of git merge.
When you use git merge, it creates "Merge branch ...... into spi-lcd" commit. Actually no need in public.
Please rebase yours and then I can merge this.